committing.rst

```diff
@@ -202,35 +202,19 @@ So a file name may be
 The contents of a news file should be valid reStructuredText. An 80 character
 column width should be used. There is no indentation or leading marker in the
 file (e.g. ``-``). There is also no need to start the entry with the issue
-number as it's part of the file name itself. Example news entry::
-
-   Fix warning message when ``os.chdir()`` fails inside
-   ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.
-
-In addition to inline reST, news entries also support the usage of
-`Sphinx roles <https://devguide.python.org/documenting/#id4>`_. Several
-commonly used roles include :func:, :class:, and :meth:. When a
-corresponding entry is found within the documentation, the text within
-the role is converted into a hyperlink. Example news entry with Sphinx::
+number as it's part of the file name itself. You can use
+:ref:`inline markups <rest-inline-markup>` too. Example news entry::
 
    Fix warning message when :func:`os.chdir` fails inside
    :func:`test.support.temp_cwd`. Patch by Chris Jerdonek.
 
-The inline Sphinx roles can be used to assist readers in finding more
-information and context on the changes made. If the inline link does not
-provide additional context, an inline reST code block can be used instead::
-
-   ``<object>``
+The inline Sphinx roles like ``:func:`` can be used to assist readers in finding
+more information. You can build html and verify that the link target is
```

Comment:
Could we add a sentence in here (just after the first one) briefly explaining what the Sphinx roles do? It would look something like this:
```
The inline Sphinx roles like ``:func:`` can be used to assist readers in finding
more information. If a matching reference is found within the
documentation, the text after the role is converted into a link.
```
I think it would be helpful to very briefly explain what the Sphinx roles actually do, since readers might not be familiar with them.

Reply:
No.  It should be documented in "documenting" chapter.  It is not necessary and sufficient.

Reply:
And "If a matching reference is found within the documentation," is not correct.  Think about `` :pep:`580` ``.

Reply:
> No. It should be documented in "documenting" chapter. It is not necessary and sufficient.

Okay, that's fine then.